pkg/didcomm/packer/authcrypt/pack.go

```diff
@@ -51,8 +51,16 @@ type Packer struct {
 // The returned Packer contains all the information required to pack and unpack payloads.
 func New(ctx packer.Provider, encAlg jose.EncAlg) (*Packer, error) {
 	k := ctx.KMS()
+	if k == nil {
+		return nil, errors.New("authcrypt: failed to create packer because KMS is empty")
```

Comment:
same here, this check doesn't make sense this deep into the framework

Reply:
same response as `anoncrypt` above